idunn/utils/categories.yml

```diff
@@ -1,144 +1,602 @@
-            - "écoles maternelles privées"
-            - "écoles primaires privées"
-            - "collèges privés"
-            - "lycées d'enseignement général et technologique privés"
-            - "enseignement supérieur public"
-            - "enseignement supérieur privé"
-        raw_filters:
-            - "school,*"
-            - "college,*"
-        regex: "^ecole|^college|^universit|educat"
+          - "banques"
+
+    fast_food:
+        raw_filters:
+          - "fast_food,*"
+        pj_what: "fast food"
+
     bar:
-        pj_what: "bars"
+        regex: "^(cafes?|bars?|pubs?)( |$)"
+        raw_filters:
+          - "bar,*"
+          - "cafe,*"
+          - "pub,*"
+          - "biergarten,*"
+        pj_what: "bar"
         pj_filters:
-            - "cafés, bars"
+          - "cafés, bars"
+
+    hotel:
+        regex: "hotel"
         raw_filters:
-            - "bar,*"
-            - "cafe,*"
-        regex: "^(cafes?|bars?|pubs?)( |$)"
-    museum:
-        pj_what: "musée"
+          - "*,hotel"
+          - "*,chalet"
+          - "*,hostel"
+          - "*,alpine_hut"
+          - "*,motel"
+          - "*,bed_and_breakfast"
+          - "*,guest_house"
+          - "*,dormitory"
+        pj_what: "hôtels"
         pj_filters:
-            - "musées"
+          - "hôtels"
+
+    historic:
         raw_filters:
-            - "museum,*"
-        regex: "^(museum|musee)"
-    health:
-        pj_what: "hopital"
+          - "memorial,*"
+          - "monument,*"
+          - "statue,*"
+          - "castle,*"
+          - "citywalls,*"
+          - "fort,*"
+          - "battlefield,*"
+          - "castle_walls,*"
+          - "earthworks,*"
+          - "moat,*"
+          - "church,*"
+          - "tumulus,*"
+          - "stone_circle,*"
+          - "menhir,*"
+          - "standing_stone,*"
+          - "bridge,*"
+          - "milestone,*"
+          - "wall,*"
+          - "well,*"
+          - "folly,*"
+          - "ruins,*"
+        pj_what: "monument historique"
+
+    post_office:
+        raw_filters:
+          - "post_office,*"
+        pj_what: "bureau de poste"
+
+    fuel:
+        raw_filters:
+          - "fuel,*"
+        pj_what: "station essence"
+
+    community_centre:
+        raw_filters:
+          - "*,community_centre"
+        pj_what: "centre communal"
+
+    shop_convenience:
+        raw_filters:
+          - "*,convenience"
+          - "*,deli"
+        pj_what: "épicerie"
+
+    shop_car:
+        raw_filters:
+          - "*,car_repair"
+          - "*,car"
+        pj_what: "concessionnaire"
+
+    kindergarten:
+        raw_filters:
+          - "*,kindergarten"
+        pj_what: "crèche"
+
+    camp_site:
+        raw_filters:
+          - "camp_site,*"
+          - "caravan_site,*"
+        pj_what: "camping"
+
+    station:
+        regex: "^gare( |$)"
+        raw_filters:
+          - "railway,station"
+          - "railway,tram_stop"
```

Comment:
Is it really relevant to include "tram_stop" here ? And why not "subway" too in this case ?  
An intention about "station" would probably be about actual train stations, don't you think ?